cltk/tests/test_tokenize_fr.py

```diff
@@ -0,0 +1,137 @@
+            "Vïande au sel de la salliere N\'atouche, c\'est laide manière.",
+            "Tes narilles fourgier ne vueilles De tes doies, ne tes oreilles."]
+        tokenizer = TokenizeSentence('french')
+        tokenized_sentences = tokenizer.tokenize_sentences(text)
+        self.assertEqual(tokenized_sentences, target)
+        assert_equal.__self__.maxDiff = None
+
+    def test_french_word_tokenizer(self):
+        """Test French-specific word tokenizer."""
+        word_tokenizer = WordTokenizer('french')
+
+        # Test sources:
+        # - Wace - St George
+        # - Marie de France - Guigemar
+        #
+
+        tests = ["Bel gent, qui venuz este ensemble "
+                 "Oïr le bien, si com moi semble, "
+                 "Le bien vos sui ci venuz dire, "
+                 "Et de saint Jorge le martyre "
+                 "Haus hom fu de noble lignage, "
+                 "Simples, pitos et sans oltrage ; "
+                 "De bones mors, de sainte vie "
+                 "Et trop pros en chevalerie. "
+                 "Uns enpereres Daciens, "
+                 "Qui haet Deu et crestiens, "
+                 "Contre nostre lei estriveit, "
+                 "Quant li bon sains Jorge viveit. "
+                 ]
+
+        results = []
+
+        for test in tests:
+            result = word_tokenizer.tokenize(test)
+            results.append(result)
+
+        target = [['Bel', 'gent', ',', 'qui', 'venuz', 'este', 'ensemble', 'Oïr', 'le', 'bien', ',', 'si',
+                   'com', 'moi', 'semble', ',', 'Le', 'bien', 'vos', 'sui', 'ci', 'venuz', 'dire', ',', 'Et',
+                   'de', 'saint', 'Jorge', 'le', 'martyre', 'Haus', 'hom', 'fu', 'de', 'noble', 'lignage', ',',
+                   'Simples', ',', 'pitos', 'et', 'sans', 'oltrage', ';', 'De', 'bones', 'mors', ',', 'de', 'sainte',
+                   'vie', 'Et', 'trop', 'pros', 'en', 'chevalerie', '.', 'Uns', 'enpereres', 'Daciens', ',',
+                   'Qui', 'haet', 'Deu', 'et', 'crestiens', ',', 'Contre', 'nostre', 'lei', 'estriveit', ',',
+                   'Quant', 'li', 'bon', 'sains', 'Jorge', 'viveit', '.']
+                  ]
+
+        self.assertEqual(results, target)
+
+    def test_nltk_tokenize_words(self):
+        """Test wrapper for NLTK's PunktLanguageVars()"""
+        tokens = nltk_tokenize_words("Sentence 1. Sentence 2.", attached_period=False)
+        target = ['Sentence', '1', '.', 'Sentence', '2', '.']
+        self.assertEqual(tokens, target)
+
+    def test_nltk_tokenize_words_attached(self):
+        """Test wrapper for NLTK's PunktLanguageVars(), returning unaltered output."""
+        tokens = nltk_tokenize_words("Sentence 1. Sentence 2.", attached_period=True)
+        target = ['Sentence', '1.', 'Sentence', '2.']
+        self.assertEqual(tokens, target)
+
+    def test_sanskrit_nltk_tokenize_words(self):
```

Comment:
Remove `test_sanskrit_nltk_tokenize_words` and `test_sanskrit_nltk_tokenize_words_attached`

Reply:
Ignore previous comment for this line; it is changed by later commits in the PR